dev/devicelab/test/adb_test.dart

```diff
@@ -5,7 +5,7 @@
 import 'dart:async';
 
 import 'package:test/test.dart';
-import 'package:collection/collection.dart';
+import 'package:collection/collection.dart' show ListEquality, MapEquality;
```

Comment:
This code runs in the Dart VM so it can't import dart:ui (which foundation does).